tests/manifest-urdna2015.jsonld

```diff
@@ -4,7 +4,7 @@
     "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
     "mf": "http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#",
     "mq": "http://www.w3.org/2001/sw/DataAccess/tests/test-query#",
-    "rdfn": "https://w3c.github.io/rch-rdc/tests/vocab#",
+    "rdfn": "https://w3c.github.io/rdf-canon/tests/vocab#",
```

Comment:
```suggestion
    "rdfn": "http://w3c.github.io/rdf-canon/tests/vocab#",
```

Reply:
I guess it doesn't really matter, but I wonder if there is some "best practice" in the RDF / JSON-LD world whether to use http or https for namespaces..